src/assets/locales/de/main.json

```
@@ -1066,6 +1066,18 @@
       "metadata_url_required_error": "Metadaten-URL ist erforderlich",
       "metadata_url_invalid_error": "Bitte geben Sie eine gültige Metadaten-URL ein"
     },
+    "clearinghouseSelfDescription": {
+      "heading": "Status der Eigenerklärung",
+      "description": "Diese Seite bietet einen Überblick über den aktuellen Stand der Selbstbeschreibungen für Administratoren.",
```

```suggestion
      "description": "Diese Seite bietet einen Überblick über den aktuellen Stand der Eigenerklärungen für Administratoren.",
```

Please search the entire file for `Selbstbeschreibung` and correct with `Eigenerklärung`

---

Changes done